packages/server-web/src/client/socket-connection.js

```diff
@@ -1,3 +1,4 @@
+import { stateToCommands } from "@todastic/storage-events";
```

Comment:
not sure what `stateToCommands` means?

Reply:
Taking a state / list of Todos and convert it to commands to create that state (`ADD_TODO` commands, actually)